basicauth_test.go

```diff
@@ -7,7 +7,7 @@ import (
 
 	"github.com/gobuffalo/buffalo"
 	"github.com/gobuffalo/httptest"
-	"github.com/gobuffalo/mw-basicauth"
+	"github.com/robsliwi/mw-basicauth"
```

Comment:
Broken import path.

Please read https://github.com/gobuffalo/buffalo/blob/master/.github/CONTRIBUTING.md on how to handle Go PRs.